src/test/java/com/github/dockerjava/core/command/ListContainersCmdImplTest.java

```diff
@@ -98,6 +97,7 @@ public void testListContainers() throws Exception {
         Container container2 = filteredContainers.get(0);
         assertThat(container2.getCommand(), not(isEmptyString()));
         assertThat(container2.getImage(), startsWith(testImage));
+        assertThat(container2.getState(), equalTo("running"));
```

Comment:
I think you will have test case error for 1.11 API version, test suite is running against multiple docker version. Add if case dependent on client api version (you can find example in some other test case)